determined-ai/hew
# refactor: convert LogViewer to React Virtuoso [WEB-1954]

Convert LogViewer to use React-Virtuoso instead of react-window. [Here](https://github.com/determined-ai/hew/pull/103) is the PR performing the same conversion on the ChartGrid component for reference. There is a companion PR in the determined repo here: https://github.com/determined-ai/determined/pull/8862.

### Test Plan

Make sure there aren’t any feature regressions after the conversion, including but not limited to:
- Tailing
- Scroll to oldest
- Download logs
- Fullscreen
- Filter logs
- Fetch logs on scroll
